### DISCUSSION TITLE
Tips about GSoC

### DISCUSSION THREAD

**Discussion by adityaraute** [EddieHubCommunity/support#536] [Q&A]
I am planning to try my hand at GSoC this year. But I am not fully sure what it is and it is gonna be my first year.
Their website does have info but it is very compressed. I would be glad to receive some info from you folks on how I should approach the event, what things to look out for, and even accounts of firsthand experiences, if any.
Hope many more pick up GSoC after this discussion.

**Comment by eddiejaoude** [comment]
Great question @dumbcoder7 . I will share in our Discord to see if people can add more information about it 👍 - I do not know much about it, but I can try to find out more also

**Comment by aeyshubh** [comment]
Google does this event once a year where students get selected to work in a organization and complete a project whichever you choose. You need to make a proposal for that organization mentioning how you would be working in the time span of 3 months . Google also give stipend for your work. MORE details can be found on official website and reading the timeline very carefully. 
Hope your doubts are clear.

**Comment by NirmalSilwal** [comment]
any more suggestions from the community members?

**Comment by bugmaker2** [comment]
I am interested in this year's GSoC too, any more information?

**Comment by bugmaker2** [comment]
Let me show you an answer  from a Q&A community in China.
You can translate it by machine. Or you can ask me for some details.
[Answer's link](https://www.zhihu.com/question/66687826/answer/375742767)
That's all I know, and I hopw it helps.

>Briefly introduce my background: CS, University of Minnesota, USA, more than one year's experience in full stack development and architecture;I participated in GSoC in 2018 and the project is Elastic Kibana
>1. select project
In the middle of February each year, Google releases GSoC org for students to study and choose projects they like or to submit to ORG projects they are interested in.
>2. Write a proposal
In early march began to write the proposal, this stage requires careful research project requirements, also can be in the org corresponding IRC, mailing list or discuss channel puts forward his own problems, through the feedback from the mentors to honing their proposal, more important, don't show you off during this technology, because your technology in mentors eyes might pale;A key step is to be clear about your ideas so he can see how you can contribute to a project
>3. Submit an application
>4. wait
Be aware that during this time, any other requests come along with Mentors and should not come easily (like adding friends on Linkedin)
>5. proposal accepted
 prepare for community bonding
>6. Three periodic evaluations
Each other between the students and mentors to submit evaluation, each stage submitted assessment of mentors and students can see each other, if mentors think you did well but there is still room for improvement you need to the next stage work harder (according to the director of GSoC Stephanie, every stage of the elimination rate at about 5%, so don't be too lax)

**Comment by eddiejaoude** [comment]
I will be discussing this with Kunal on Monday. Who has done GSOC and MLH (as a fellow and a pod leader). Bring your questions and lets discuss this and more... https://www.youtube.com/watch?v=BQXJMEkAtAQ

**Reply by adityaraute** [reply]
Excited about this one

**Reply by eddiejaoude** [reply]
Thanks, not long to go 👍

**Reply by bugmaker2** [reply]
Thanks！
